mdpow/workflows/dihedrals.py

```diff
@@ -157,18 +167,35 @@ def rdkit_conversion(u, resname):
 
     """
 
-    try:
-        solute = u.select_atoms(f'resname {resname}')
-        mol = solute.convert_to('RDKIT')
-    except AttributeError:
-        elements = [guess_atom_element(name) for name in u.atoms.names]
-        u.add_TopologyAttr("elements", elements)
-        solute = u.select_atoms(f'resname {resname}')
-        mol = solute.convert_to('RDKIT')
+    # i think somewhere we decided to ditch try/except method, will check notes
+    # notes pertaining to testing topology attributes for explicit Hs
```

Comment:
I don't remember. 

But document in the docstring that we _always_ guess elements using the MDA guesser.